### PR TITLE
[16.0][MIG] delivery_total_weight_from_packaging: Migrate to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Depend on:
- https://github.com/OCA/product-attribute/pull/1404

### CODE REVIEW

**Review by cyrilmanuel — APPROVED** [review]
not an expert but LGTM

**Review by sebalix — APPROVED** [review]
LG, last commit `[DON'T MERGE] test-requirements.txt` to drop

**Comment by QuocDuong1306** [comment]
Thank to everybody, done here

File: delivery_total_weight_from_packaging/tests/test_weight_from_packaging.py [hunk 1]
```diff
@@ -49,13 +49,13 @@ def test_picking_shipping_weight(self):
...
         for line in picking.move_line_ids:
-            line.qty_done = line.product_qty
+            line.qty_done = line.reserved_uom_qty
         self.assertTrue(picking.package_ids)
         self.assertTrue(picking.move_ids_without_package)
         # Check shipping weight knowing there is no shipping weight on the package
         self.assertEqual(picking.shipping_weight, 16)
         pack.shipping_weight = 6
-        picking.invalidate_cache(["shipping_weight"])
+        picking._invalidate_cache(["shipping_weight"])
```
Comment (rousseldenis) [inline delivery_total_weight_from_packaging/tests/test_weight_from_packaging.py]: @QuocDuong1306 Is it still necessary ?

Comment (QuocDuong1306) [reply delivery_total_weight_from_packaging/tests/test_weight_from_packaging.py]: Thank @rousseldenis , updated

**Review by mmequignon — APPROVED** [review]

**Review by cyrilmanuel — APPROVED** [review]

**Pull request merged**